kong/init.lua

```diff
@@ -1570,6 +1570,9 @@ end
 
 
 function Kong.serve_wrpc_listener(options)
+  if kong.configuration.legacy_hybrid_protocol then
+    return ngx.exit(404)
```

Comment:
instead of return 404 here, we can also skip rendering the wrpc server block in nginx_template